4. Funciones/Ejercicio 4.py

```diff
@@ -0,0 +1,29 @@
+"""Haga una función que retorne el número de veces que un número aleatorio del 1 al 10 aparezca en 
+otro número aleatorio de 10000. Replique la función utilizando Lambda."""
+
+
+import random
+
+#Función normal
+
+rand_number = random.randint(1,10)
+rand_big = random.randint(1,10000)
+
+def frequency (x):
```

Comment:
This is wrongly implemented.

Reply:
No entiendo qué se debe implementar... Defino un número aleatorio del 1 al 10, luego defino otro número aleatorio de 10000, ¿qué hay que hacer ahí? ¿Contar las veces que sale en el string?